### PR TITLE
Updated readme, added testing with Seneca@1.4.0 and 2.0.x

### PR DESCRIPTION
**Pull request by shanel262**

### CODE REVIEW

**Comment by shanel262** [comment]
#11

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 [![Gitter chat][gitter-badge]][gitter-url]
 
 This plugin is included with the main seneca module and provides a
 small set of basic utility action patterns.
 
 - __Version:__ 0.3.0
-- __Tested on:__ [Seneca](https://github.com/senecajs/seneca) 0.7
+- __Tested on:__ [Seneca](https://github.com/senecajs/seneca) 1.4.0
 - __Node:__ 0.10, 0.12, 4
 - __License__ [MIT][]
```
Comment (mihaidma) [inline README.md]: please update readme using as model the https://github.com/senecajs/seneca-auth one

File: README.md [hunk 2]
```diff
@@ -155,4 +155,4 @@ Copyright Richard Rodger and other contributors 2015, Licensed under [MIT][].
 [senecajs.org]: http://senecajs.org/
 [github issue]: https://github.com/senecajs/seneca-basic/issues
 [@senecajs]: http://twitter.com/senecajs
-[seneca-admin]: https://github.com/rjrodger/seneca-admin
\ No newline at end of file
+[seneca-admin]: https://github.com/rjrodger/seneca-admin
```
Comment (mihaidma) [inline README.md]: link needs update

Comment (shanel262) [inline README.md]: @mihaidma This repo is still on rjrodger

File: README.md [hunk 3]
```diff
@@ -134,25 +125,41 @@ _Response:_
...
+
+```sh
+npm run test
+```
+
 ## License
-Copyright Richard Rodger and other contributors 2015, Licensed under [MIT][].
+
+Copyright (c) 2014, Richard Rodger and other contributors.
+Licensed under [MIT][].
```
Comment (mihaidma) [inline README.md]: update copyright 2014-2016, same on license file

**Comment by mihaidma** [comment]
LGTM, ready for merge after minor fixes

File: README.md [hunk 4]
```diff
@@ -1,17 +1,16 @@
...
 [![Gitter chat][gitter-badge]][gitter-url]
 
 This plugin is included with the main seneca module and provides a
 small set of basic utility action patterns.
 
-- __Version:__ 0.3.0
-- __Tested on:__ [Seneca](https://github.com/senecajs/seneca) 0.7
-- __Node:__ 0.10, 0.12, 4
-- __License__ [MIT][]
+- __Node:__ 0.10, 0.12, 4, 5
```
Comment (mihaidma) [inline README.md]: please add Seneca versions

**Pull request merged**